clients/web/src/lib/components/atoms/form/Checkbox/Checkbox.svelte

```diff
@@ -0,0 +1,63 @@
+<script lang="ts">
+    import {slide} from "svelte/transition";
+    import {Check} from "@steeze-ui/heroicons";
+    import {Icon} from "@steeze-ui/svelte-icon";
+    import {nanoid} from "nanoid";
+    import type {FormControlState} from "../form.types";
```

Comment:
Lets just use `types.ts` instead of `atom/atom.types.ts`; fuzzy finding for files is good enough that it'll find what you're looking for; and barrel files will hide the file outside the atom directory anyways